patterns/cta.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Title: Centered Call To Action
+ * Slug: twentytwentyfour/cta
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"","padding":{"top":"100px","right":"100px","bottom":"100px","left":"100px"}}},"backgroundColor":"base","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-base-background-color has-background" style="padding-top:100px;padding-right:100px;padding-bottom:100px;padding-left:100px"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"50px","bottom":"50px"}}},"backgroundColor":"tertiary","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-tertiary-background-color has-background" style="padding-top:50px;padding-bottom:50px"><!-- wp:paragraph {"align":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"typography":{"fontSize":"28px","lineHeight":"1.6"}},"textColor":"contrast","fontFamily":"cardo"} -->
+<p class="has-text-align-center has-contrast-color has-text-color has-link-color has-cardo-font-family" style="font-size:28px;line-height:1.6">Join 900+ designers</p>

```

Comment:
Strings need to be made translateable with the `esc_html__` or similar functions. You can look at other patterns how to do that.